packages/veui/src/utils/context.js

```diff
@@ -100,3 +100,26 @@ export function getIndexOfType (current, type, vnodes = 'default') {
     vnode => vnode === currentVNode
   )
 }
+
+/**
+ * 查找该组件的子孙组件中第一个有某个方法的组件实例，深度优先
+ * @param {Vue} context 查找的组件实例，即从 context.$children 开始查找
+ * @param {(vm: Vue) => boolean} predicate 判断这个组件是否匹配
```

Comment:
https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#function-type

`function(Vue): boolean`